browser-test/src/support/index.ts

```diff
@@ -383,7 +383,7 @@ async function loginAsTestUserAwsStaging(
   await page.fill('input[name=username]', TEST_USER_LOGIN)
   await page.fill('input[name=password]', TEST_USER_PASSWORD)
   await Promise.all([
-    page.waitForURL(isTi ? '**/admin/**' : '.*/programs.*', {
+    page.waitForURL(isTi ? '**/admin/**' : /.*\/programs.*/, {
```

Comment:
The `\` is to escape the `/` before `programs` right?

Reply:
Correct!